test/components/fa-icon.test.js

```diff
@@ -0,0 +1,30 @@
+const numbersIcon = require("../../src/components/fa-icons/numbers-icon")
+const asterisk = require("../../src/components/fa-icons/asterisk-icon")
+const clearAll = require("../../src/components/fa-icons/clear-all-icon")
+const delet = require("../../src/components/fa-icons/delete-icon")
+const devide = require("../../src/components/fa-icons/divide-icon")
+const equals = require("../../src/components/fa-icons/equals-icon")
+const minus = require("../../src/components/fa-icons/minus-icon")
+const percent = require("../../src/components/fa-icons/percent-icon")
+const period = require("../../src/components/fa-icons/period-icon")
+const plus = require("../../src/components/fa-icons/plus-icon")
+const plusMinus = require("../../src/components/fa-icons/plus-minus-icon")
+
+describe("Font Awesome icons", () =>{
```

Comment:
but i was thinking about the thing you've said, about "if someone would change the icon's font to another".
because the font awesome have one class for them icons => fa-c;fa-plus
and for example Tabler have another, like ti-c; ti-plus